.github/workflows/weekly-resources-clean.yml

```diff
@@ -31,7 +31,8 @@ jobs:
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
           aws-region: us-west-2
 
-      - name: Clean old ec2
-        run: go run tools/workflow/clean_ec2.go
+      - name: Clean old aws resources
+        run: go run tools/workflow/clean_aws_rsc.go
```

Comment:
Leave it as it you don't agree but I would prefer not to use rsc and just spell it out

Reply:
I agree, but I'm nitpicky. `rsc` looks like an acronym.

Reply:
Agreed. Not using acronym anymore lol.